README.md

```diff
@@ -375,4 +377,4 @@ Google's Project Zero series of articles that detail vulnerabilities in the wire
 * [@patrickolsen](https://twitter.com/patrickolsen)
 
 ## Other OSX Awesome lists
-* [ashishb/osx-and-ios-security-awesome](https://github.com/ashishb/osx-and-ios-security-awesome)
\ No newline at end of file
+* [ashishb/osx-and-ios-security-awesome](https://github.com/ashishb/osx-and-ios-security-awesome)
```

Comment:
Not sure how this change creeped into the diff? I used the Web-based GitHub editor to submit the PR.

Reply:
Looks like your editor automatically removed the whitespace at the end of the line. No big deal